src/resources/CronJobs/CronJobDescription.js

```diff
@@ -0,0 +1,11 @@
+import { Trans } from 'react-i18next';
+import { Link } from '../../shared/components/Link/Link';
+
+export const description = (
```

Comment:
This file is not used. Leave only one file with description

Reply:
Adjusted